### PR TITLE
Main copier update

### PR DESCRIPTION
**Pull request by nvaytet**

### CODE REVIEW

File: .copier-answers.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changes here will be overwritten by Copier; NEVER EDIT MANUALLY
+_commit: 6770edb
+_src_path: gh:scipp/copier_template
+description: Common data reduction tools for the ESS facility
+max_python: '3.11'
```
Comment (SimonHeybrock) [inline .copier-answers.yml:5]: Why not 3.12?

File: .copier-answers.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+min_python: '3.10'
+namespace_package: ess
+nightly_deps: scipp,sciline
```
Comment (SimonHeybrock) [inline .copier-answers.yml:8]: We do not have dependencies here, for now, remove?

File: .copier-answers.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+orgname: scipp
+prettyname: ESSreduce
+projectname: essreduce
+related_projects: ESSsans,ESSdiffraction,ESSreflectometry,ESSnmx,ESSimaging,ESSspectroscopy
```
Comment (SimonHeybrock) [inline .copier-answers.yml:12]: ESSpolarization

File: pyproject.toml [hunk 4]
```diff
@@ -29,6 +29,7 @@ requires-python = ">=3.10"
 # Run 'tox -e deps' after making changes here. This will update requirement files.
 # Make sure to list one dependency per line.
 dependencies = [
+    "scipp",
```
Comment (SimonHeybrock) [inline pyproject.toml:32]: Why?

**Comment by nvaytet** [comment]
Closing in favour of #4 (was easier to start again)

**Pull request closed without merging**